tests/main/views/test_drafts.py

```diff
@@ -1085,6 +1140,17 @@ def test_should_not_copy_draft_documents(self):
         assert "serviceDefinitionDocumentURL" not in data['services']
         assert "sfiaRateDocumentURL" not in data['services']
 
+    @mock.patch('app.db.session.commit')
+    def test_copy_draft_service_should_catch_db_integrity_error(self, db_commit):
+        db_commit.side_effect = IntegrityError("Could not commit", orig=None, params={})
+        res = self.client.post(
+            '/draft-services/%s/copy' % self.draft_id,
```

Comment:
I was wondering about that - one of the existing tests for this endpoint used `%` so I was going for 'consistency'. I can change the old test too 😄